rezd10-sp2

rezacdom opened the pull request:
Link to application: https://eso.vse.cz/~rezd10/sp2/

rezacdom committed:
rezd10-sp2

Link to application: https://eso.vse.cz/~rezd10/sp2/

rezacdom committed:
Verze 2 - finální

Finální verze projektu, dopilování veškerých nedokonalostí z původní veze.

rezacdom committed:
Delete www/REZD10/sp2/style.css

Smazání původního css, které bylo ve finální verzi přemístěno do správné složky.

rezacdom committed:
Update index.html

Opravena cesta k css

nvbach91 reviewed (commented):
- vyber elementu z HTML provest pomoci jQuery
- elementy z HTML nevybirat opakovane v prubehu programu, ale vybirat pouze jednou a ukladat je do promennych
- API BASE_URL ukladat do konstanty
- nezasahovat do DOM v cyklu (pridavani novych elementu, zmena atributu, obsahu, atd.)
- funkce deklarovat jako lokalni pomoci `const`
- data z localStorage nenacitat opakovane ale nacist pouze jednou na zacatku programu a drzet je v in-memory objektu, jehoz obsah se pak bude menit a ukladat do localStorage
- nepouzivat inline javascript
- nevykreslovat cely obsah znova po zadani hodnoceni, pridani do oblibenych, odebirani z oblibenych, ...

rezacdom commented:
Dobrý den,
@nvbach91
Muzu jeste do zitrejsi obhajoby opravit, nebo uz ne?
Musi byt aplikace kompatibilni s mobilnimi zarizenimi?
Dekuji za odpoved,
Řezáč

rezacdom committed:
Oprava chyb z komentáře

oprava chyb z komentáře:
vyber elementu z HTML provest pomoci jQuery
elementy z HTML nevybirat opakovane v prubehu programu, ale vybirat pouze jednou a ukladat je do promennych
API BASE_URL ukladat do konstanty
nezasahovat do DOM v cyklu (pridavani novych elementu, zmena atributu, obsahu, atd.)
funkce deklarovat jako lokalni pomoci const
data z localStorage nenacitat opakovane ale nacist pouze jednou na zacatku programu a drzet je v in-memory objektu, jehoz obsah se pak bude menit a ukladat do localStorage
nepouzivat inline javascript
nevykreslovat cely obsah znova po zadani hodnoceni, pridani do oblibenych, odebirani z oblibenych, ...
+
oprava zobrazení pro telefony
drobné opravy

nvbach91 commented:
- zobrazit detaily filmu

rezacdom committed:
Oprava z obhajoby

Opraveny chyby:
vyber elementu z HTML provest pomoci jQuery
elementy z HTML nevybirat opakovane v prubehu programu, ale vybirat pouze jednou a ukladat je do promennych
API BASE_URL ukladat do konstanty
nezasahovat do DOM v cyklu (pridavani novych elementu, zmena atributu, obsahu, atd.)
funkce deklarovat jako lokalni pomoci const
data z localStorage nenacitat opakovane ale nacist pouze jednou na zacatku programu a drzet je v in-memory objektu, jehoz obsah se pak bude menit a ukladat do localStorage
nepouzivat inline javascript
nevykreslovat cely obsah znova po zadani hodnoceni, pridani do oblibenych, odebirani z oblibenych, ...
+
Zobrazení detailu filmu

rezacdom committed:
Oprava připomínek

Oprava připomínek z opravy na obhajobě
